proto/stream_service.proto

```diff
@@ -14,7 +14,7 @@ message InjectBarrierRequest {
   stream_plan.Barrier barrier = 2;
   repeated uint32 actor_ids_to_collect = 4;
   repeated uint32 table_ids_to_sync = 5;
-  uint32 partial_graph_id = 6;
+  uint64 partial_graph_id = 6;
```

Comment:
I actually prefer the `partial_graph_id` to be represented as a struct, in the type system, to let compiler do necessary checks for us.

We can also easily understand the components of the id, which are `database_id, job_id`

Reply:
I used a structural `partial_graph_id` at the beginning. But soon I realized that in the future when we support partial checkpoint, a partial graphs may include multiple jobs in a single database, and it will be hard to represent it in a structural way. Therefore I changed to use a meaningless u64 to represent the `partial_graph_id`, and the CN side will not be aware of the meaning of a `partial_graph_id`. Meta node will instead maintain the mapping of `partial_graph_id` to the partial graph.

Reply:
Got your point.